src/pages/host/components/HostReviews.tsx

```diff
@@ -47,58 +43,64 @@ function HostReviews({ host }: { host: HostProfile }) {
           </IconButton>
         </Box>
       </Box>
-      <Box>
-        <Slider
-          itemsPerView={itemsPerView}
-          activeStep={activeStep}
-          onStepChange={setActiveStep}
-          maxSteps={maxSteps}
-          onEmpty={
-            <Typography key="error" textAlign="center">
-              {t(HostInfo.host_reviews_empty)}
-            </Typography>
-          }
-        >
-          {reviews.list?.map((review) => (
-            <Box
-              key={review.id}
-              sx={{
-                height: '100%',
-                display: 'flex',
-                flexDirection: 'column',
-                borderRadius: 5,
-                border: '1px solid #b0b0b0',
-                p: 3,
-              }}
-            >
-              <Box textOverflow={'ellipsis'} height="9rem" mb={4} overflow="auto">
-                <Typography flex={1} fontSize="1.1rem">
-                  {review.feedback}
+
+      <Slider
+        itemsPerView={itemsPerView}
+        activeStep={activeStep}
+        onStepChange={setActiveStep}
+        maxSteps={maxSteps}
+        onEmpty={
+          <Typography key="error" textAlign="center">
+            {t(HostInfo.host_reviews_empty)}
+          </Typography>
+        }
+      >
+        {reviews.list?.map((review) => (
```

Comment:
Looks like you need to create Reviews list or ReviewListItem component created to make the code more structured